akka-stream-tests/src/test/scala/akka/stream/io/InputStreamSourceSpec.scala

```diff
@@ -3,17 +3,18 @@
  */
 package akka.stream.io
 
-import java.io.InputStream
+import java.io.{ OutputStream, PipedOutputStream, PipedInputStream, InputStream }
 import java.util.concurrent.CountDownLatch
 
-import akka.stream.scaladsl.Sink
-import akka.stream.scaladsl.StreamConverters
+import akka.stream.scaladsl.{ Source, Sink, StreamConverters }
 import akka.stream.testkit._
 import akka.stream.testkit.Utils._
 import akka.stream.testkit.scaladsl.TestSink
-import akka.stream.{ ActorMaterializer, ActorMaterializerSettings }
+import akka.stream.{ Attributes, OverflowStrategy, ActorMaterializer, ActorMaterializerSettings }
 import akka.util.ByteString
 import org.scalatest.concurrent.ScalaFutures
+import scala.concurrent.Await
+import scala.concurrent.duration._
```

Comment:
It could be via patience config but this is OK I believe (need to await longer than I allow it to run)